Cargo.lock

```diff
@@ -3760,6 +3760,7 @@ dependencies = [
 name = "query-builder"
 version = "0.1.0"
 dependencies = [
+ "quaint",
```

Comment:
I'm not sure if it's good for `query-builder` to depend on `quaint`, at least from the architecture point of view. Is there a better place for the fragment types so that both `quaint` and `query-builder` would get import them from there?

Reply:
Maybe we should make `query_template` a crate rather than a module in quaint?

Reply:
It does not belong into `query-structure`, so a separate crate then.